tanjun/context.py

```diff
@@ -1381,3 +1391,5 @@ async def respond(
 
         if ensure_result:
             return await self._interaction.fetch_initial_response()
+        else:
+            return None
```

Comment:
Don't really want a ton of random out-of-place `return None` statements around just cause mypy has issues tbh